source/manual/howto-backup-and-restore-in-aws-rds.html.md

```diff
@@ -230,13 +230,26 @@ This requires updating the CNAME for `local-links-manager-postgres`.
 4. Replace the value with the new RDS backup and save your changes. This takes about 60 seconds, you can click "view status" for updates. Once updated it will say `INSYNC`.
 5. SSH back into the machine and query for the record you deleted. If the record is back this should verify the app is now using the backup database.
 
-### 3. Connect back to the original database
+> This is only a temporary solution, to be used in an incident. You should continue onto the next section for a permanent solution.
+
+## Ensure your setup will continue to work if infrastructure is reprovisioned
+
+If new infrastructure is provisioned, then the "Point the application to the new RDS instance" solution above will break, as Terraform would fall out of sync with the manual changes. We either need to update Terraform with our changes, or manually get our infrastructure back to how it used to be.
```

Comment:
Should we try and think of a process to block deploying certain terraform projects? Is it hard to state here which ones they might be? (I guess different apps might have a different set of deps)

Reply:
Probably better/easier to make the Terraform changes rather than build in a process for blocking deployments.